pkg/manifests/bundleloader.go

```diff
@@ -38,6 +39,15 @@ func (b *bundleLoader) LoadBundle() error {
 
 	errs = append(errs, b.calculateCompressedBundleSize())
 
+	// Set values from the annotations when the values are not loaded
```

Comment:
This seems fine, but do we document that channels/default channels set in the bundle are prioritized over those set in the bundle?

Reply:
Currently, it has not been populated in any way. The default and channels are never populated and that was the issue that we are fixing here. We actually do not need the do the check conditional here.

Reply:
Added a comment to clarify it in the code.